notes/transcript11.md

```diff
@@ -0,0 +1,585 @@
+# Intro
+
+0:00 - 1:00
+
+Oskar: Welcome to the eleventh Status Core Dev Call. We have quite a lot of things on the agenda today. We did a poll SMT voting to find the things people want to talk the most about. We'll start from the top and see how far we get with the time we have. Also just a small PSA, if you go to the repository, status-im/pm, we have a index of all the previous videos and notes if you want to refer back. Let's get started, so the first topic is Whisper availability inside Status Dapp browser, which is something that will benefit Teller Network as well as Gas Relayer and possibly others. So I guess Richard, would like to elaborate on it if you don't mind?
+
+
+# Whisper Availability
+
+1:00 - 2:00
+
+Richard: So basically we are going to do the development of the Gas Relayer and Teller Network. We saw that we have a need to have Whisper available in all the apps, mostly because we want all the apps to be able to communicate between each other. What the rest of the team working on the Gas Relayer and Teller Network and I wanted to know is what is the coordinate state of this functionality and what would it take for Dutch to be able to use Whisper in the (Work Tree?) provider that the Status provides?
+
+Status (Dutch?): Well since I've been discussing this before, probably since it's building, I have improved my answer. So first question is, do you need this functionality only when the Dapp is open in the status browser or do you need it to send the synchronous notifications to somewhere.
```

Comment:
Status -> Igor (and below), since he was hosting the Zoom call

Reply:
fixed